src/wallet/rpcwallet.cpp

```diff
@@ -2431,7 +2443,7 @@ UniValue fundrawtransaction(const UniValue& params, bool fHelp)
 
         if (options.exists("feeRate"))
         {
-            feeRate = CFeeRate(options["feeRate"].get_real());
```

Comment:
On the other hand, imagine if someone noticed it while reviewing #7967: The pull probably would have never made it into 0.13 before feature freeze, because adding the anytype is a non-trivial refactor and review is hard to do when there is substantial refactoring going on while adding new features.

Looking at it from this perspective, it makes sense to have two pulls to aid review.